mlx/backend/metal/kernels/quantized.h

```diff
@@ -2007,6 +2011,284 @@ template <
       w, scales, biases, x, y, Xs, Ws, K, N, M, tid, lid, simd_gid, simd_lid);
 }
 
+template <typename T, typename mma_t, typename loader_a_t, typename loader_b_t>
+inline void gemm_loop_aligned(
```

Comment:
```suggestion
METAL_FUNC void gemm_loop_aligned(
```
For consistency, there shouldn't be any performance implications to this